js/interface.js

```diff
@@ -890,29 +876,26 @@ $('#app')
   .on('click', '[data-show-all-source]', function() {
     $('[data-show-all-source]').addClass('hidden');
     $('[data-app-source]').removeClass('hidden');
+
     if ($('[data-show-trash-source]').hasClass('active-source')) {
       isShowingAll = false;
       $('[data-show-trash-source]').click();
     } else {
       isShowingAll = true;
 
-      var orderedDataSources = sortDataSources('updatedAt', 'desc', dataSources);
```

Comment:
Yes i removed this because i added the call of function getDataSource where we have the sorting
![изображение](https://user-images.githubusercontent.com/71251809/99401700-25b85980-28f1-11eb-998d-8870dfd69a6f.png)